models/User.js

```diff
@@ -5,7 +5,7 @@ const EMAIL_REGEX = /^([\w\.\-]+)@([\w\-]+)((\.(\w){2,3})+)$/;
 const userSchema = new Schema({
     first_name: {
         type: String,
-        required: true,
+        require: true,
```

Comment:
So at the final is it correct require or required ?

Reply:
I think required .